src/main/java/nextstep/subway/dto/LineResponse.java

```diff
@@ -25,7 +25,7 @@ private LineResponse(Long id, Name name, Color color, List<StationResponse> stat
 
     public static LineResponse from(Line line) {
         return new LineResponse(line.getId(), line.getName(), line.getColor(),
```

Comment:
dto 이기 때문에 크게 상관은 없지만, 이번 미션에서 객체지향 생활 체조 원칙을 강한 규칙으로 적용해보는 의미도 있고, 개인적으로는 아래와 같은 방식으로 작성하곤 하는데 공유차원에서 말씀드립니다 :)

```suggestion
        return new LineResponse(
                line.getId(),
                line.getName(),
                line.getColor(),
                line.findInOrderStations()
                        .stream()
                        .map(StationResponse::from)
                        .collect(Collectors.toList()),
                line.getDistance()
        );
```

Reply:
stream() 같은 체이닝일때만 .이 한줄에 하나인줄 알았는데, 이렇게 파라미터로 들어가는 경우도 .을 한줄에 하나 넣는 것이었군요! 다음 과제부터는 이부분도 체크하면서 진행하겠습니다 ㅎㅎ

피드백 감사합니다 :)

Reply:
규칙을 맞춰 작성한다기 보다는 가독성이 더 좋아보이는 방법을 사용하시면 될 것 같습니다 :) 물론 이 부분은 개인마다 의견이 다르고 팀 마다도 다르니 판단해서 적용해주세요 ㅎㅎ

고생 많으셨습니다 !! 👍👍